Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM python:3.5-slim-stretch
+FROM python:3.5-slim-buster
 
 # ensure unoconv can locate the uno library
-ENV PYTHONPATH=/usr/lib/python3/dist-packages
+ENV PYTHONPATH /usr/lib/python3/dist-packages
 
 RUN usermod -d /home www-data \
     && chown www-data:www-data /home \
+    # -slim images strip man dirs, but java won't install unless this dir exists.
+    && mkdir -p /usr/share/man/man1 \
```

Comment:
Yep, I searched a little bit and discovered that this was and still is bug for jre on slim debian. And I learned the `-p` option for `mkdir`:

> -p, --parents: no error if existing, make parent directories as needed